gratipay/billing/payday.py

```diff
@@ -27,7 +27,7 @@
 
 
 with open('sql/payday.sql') as f:
-    PAYDAY = f.read()
+    PAYDAY = f.read() % {'minimum_charge': MINIMUM_CHARGE}
```

Comment:
I think having it as a permanent table makes sense - If we start to write heavy SQL outside of payday, it could be useful.